packages/directory/config/ember-try.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const getChannelURL = require('ember-source-channel-url');
+
+module.exports = function() {
+  return Promise.all([getChannelURL('release'), getChannelURL('beta'), getChannelURL('canary')]).then(urls => {
+    return {
+      scenarios: [
+        {
+          name: 'ember-lts-2.18',
+          env: {
+            EMBER_OPTIONAL_FEATURES: JSON.stringify({ 'jquery-integration': true })
+          },
+          npm: {
+            devDependencies: {
+              '@ember/jquery': '^0.5.1',
+              'ember-source': '~2.18.0'
+            }
+          }
+        },
+        {
+          name: 'ember-lts-3.4',
+          npm: {
+            devDependencies: {
+              'ember-source': '~3.4.0'
+            }
+          }
+        },
+        {
+          name: 'ember-release',
+          npm: {
+            devDependencies: {
+              'ember-source': urls[0]
+            }
+          }
+        },
+        {
+          name: 'ember-beta',
+          npm: {
+            devDependencies: {
+              'ember-source': urls[1]
+            }
+          }
+        },
+        {
+          name: 'ember-canary',
+          npm: {
+            devDependencies: {
+              'ember-source': urls[2]
+            }
+          }
+        },
+        // The default `.travis.yml` runs this scenario via `npm test`,
+        // not via `ember try`. It's still included here so that running
+        // `ember try:each` manually or from a customized CI config will run it
+        // along with all the other scenarios.
+        {
+          name: 'ember-default',
+          npm: {
+            devDependencies: {}
+          }
+        },
+        {
+          name: 'ember-default-with-jquery',
+          env: {
+            EMBER_OPTIONAL_FEATURES: JSON.stringify({
+              'jquery-integration': true
+            })
+          },
+          npm: {
+            devDependencies: {
+              '@ember/jquery': '^0.5.1'
+            }
+          }
+        }
+      ]
+    };
+  });
+};
```

Comment:
File not used.

Reply:
do we need the `ember-try` dependency then? a standard addon has a `"test:all": "ember try:each"` npm script

Reply:
hmm, not sure.  The tests are being run by lerna via the top level package.json.  I don't think we are currently using ember-try in our tests.

Reply:
I kind of think we should keep these things in place even if we don't use them.  Ember upgrades will just put them back, and nott worth the effort to remember to remove them. let's reduce what needs to be done with each upgrade.

Reply:
I tried it with core and it works fine.  Reverting for now.

Reply:
I agree with @cythrawll that it makes sense to keep these

Reply:
In general, if we don't want a generated file we should add an entry for it in the gitignore file. 

In this case, this file can be helpful for future upgrades in determining supported Ember versions.